addon/components/frost-button.js

```diff
@@ -212,20 +212,35 @@ export default Component.extend(PropTypeMixin, {
 
   // == Events ================================================================
 
+  init () {
```

Comment:
Confirmed in this twiddle https://ember-twiddle.com/516529ad06c87d185ae98c89e396ace1?openFiles=mixins.first.js%2Ctemplates.components.my-component.hbs that we can just order a proxy mixin like this to the right of the PropTypes mixin and it'll work fine

So this would be something like `Component.extend(PropTypesMixin, HashProxyMixin`